### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by CannonLock**
Repairs all broken links on the CHTC website and implements a link checking action that is run on all PR's.

### CODE REVIEW

**Comment by CannonLock** [comment]
Most of these commits had clear link fixes but a couple commits I used my discretion so someone should check those commits over. 

- https://github.com/CHTC/chtc-website-source/pull/354/commits/b7a323eae510fc602094e13f44d66bc98f6a5b00
- https://github.com/CHTC/chtc-website-source/pull/354/commits/b30d029ae7a3e252357bafafc9e1478d262b308b

File: uw-research-computing/archived/chtcjobwrapper.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ Required. The executable invoked by the `chtcjobwrapper` script.
 -   `--cmdtorun=compiledmatlabprogram`\
     For Matlab jobs, this is your compiled Matlab program.
     [Documentation on compiling your Matlab program is available
-    here.](MATLABandR)
+    here.](MATLABandR.html)
```
Comment (ChristinaLK) [inline uw-research-computing/archived/chtcjobwrapper.md:163]:
```suggestion
    here.]()
```

Comment (ChristinaLK) [reply uw-research-computing/archived/chtcjobwrapper.md:163]: guide no longer exists, better to leave this blank.

File: uw-research-computing/archived/DAGenv.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Grid and national <a href="https://osg-htc.org/about/open_science_pool/">OS Pool
     **For Matlab, Python, and R jobs, it is first essential to have
     completed the steps to compile your code dependencies, according to
     our [\"Compiling Matlab, Python and R Code\"
-    page.](MATLABandR)**
+    page.](MATLABandR.html)**
```
Comment (ChristinaLK) [inline uw-research-computing/archived/DAGenv.md:51]:
```suggestion
    page.]()**
```

File: uw-research-computing/condor_q.md [hunk 3]
```diff
@@ -328,10 +327,10 @@ One common use of constraints is to find all jobs that are running,
 held, or idle. To do this, use a constraint with the `JobStatus`
 attribute and the appropriate status number - the status codes can be
 found in [Appendix
-A](http://research.cs.wisc.edu/htcondor/manual/current/12_Appendix_A.html)
+A](https://htcondor.readthedocs.io/en/latest/codes-other-values/job-event-log-codes.html?highlight=status%20codes)
```
Comment (ChristinaLK) [inline uw-research-computing/condor_q.md:330]:
```suggestion
A](https://htcondor.readthedocs.io/en/latest/classad-attributes/job-classad-attributes.html?highlight=JobStatus)
```

File: uw-research-computing/howto_overview.md [hunk 4]
```diff
@@ -20,27 +20,27 @@ title: CHTC Tools for Matlab, R, and Python Portability
...
 				</div>
 			</div>
 		</div>
 		<div class="uw-card">
 			<div class="uw-card-content">
-				<a href="{{ '/uw-research-computing/r-jobs' | relative_url }}"><img src="{{ '/images/R_Logo.png' | relative_url }}" alt="R logo"></a>
+				<a href="{{ '/uw-research-computing/r-jobs.html' | relative_url }}"><img src="{{ '/images/R_Logo.png' | relative_url }}" alt="R logo"></a>
 				<div class="uw-card-copy">
-					<h2 class="uw-mini-bar"><a href="{{ '/uw-research-computing/r-jobs' | relative_url }}">Running R Job in CHTC</a></h2>
+					<h2 class="uw-mini-bar"><a href="{{ '/uw-research-computing/r-jobs.html' | relative_url }}">Running R Job in CHTC</a></h2>
```
Comment (ChristinaLK) [inline uw-research-computing/howto_overview.md:43]:
```suggestion
					<h2 class="uw-mini-bar"><a href="{{ '/uw-research-computing/r-jobs.html' | relative_url }}">Running R Jobs in CHTC</a></h2>
```

**Review by ChristinaLK — APPROVED** [review]
LGTM

**Comment by ChristinaLK** [comment]
The second commit you asked me to look at is fine but isn't included in this branch/PR.

**Pull request merged**